invites/async/canReply.js

```diff
@@ -1,10 +1,11 @@
 module.exports = function (server) {
   return function canReply (invite, callback) {
-    const { author, recipient } = invite
+    const { value: { author, content: { recps } } } = invite
     server.whoami((err, whoami) => {
-      var bool = (recipient === whoami.id) &&
-        (recipient !== author)
-      callback(bool)
+      let recipients = recps.filter(recp => recp !== whoami.id)
```

Comment:
heads up in future you can skip `whoami` and use `server.id` which is just a property pinned to the server!

Reply:
that would change this into a `sync` method, which could be a good thing, except `scuttle-inject` converts sync methods into methods which return observeables ... can discuss why some other time if interested

Reply:
:white_check_mark:  changed all whoami to server.id